### ISSUE TITLE
Abgeschlossene Aufträge verschwinden nach Neustart

### ISSUE THREAD

**Issue by Oberlump** [Mmtrx/FS22_BetterContracts#246]
Ich weiss nicht, ob das ein Better Contracts Problem ist, oder ein generelles LS22-Problem:

Nach einem Neustart des Spiels verschwinden nach einigen Augenblicken die schon vor dem schließen des Spiels fertiggestellten Aufträge (Feld 24) inklusive der darin gemieteten Fahrzeuge. Manchmal tritt das Problem direkt nach dem starten des Spielstandes auf, manchmal erst nach kurzer Zeit.

Screenshot 1: Direkt nach Start des Spielstandes, Fahrzeuge auf Feld 77 noch da, Feld 24 Status noch da
Screenshot 2: Auftrag Feld 24 "Abgeschlossen" noch da.
Screenshot 3: Wenige Sekunden später: Fahrzeuge weg (nur eigener Anhänger noch da) und Feld 24 Status verschwunden / Feldstatus zurückgesetzt

Es wurde auch kein Geld des Auftrages gutgeschrieben. Hier der Eintrag aus der missions.xml zum Feld 24:

**Comment by Mmtrx** [comment]
Moin,
ich denke, das ist ein Problem des base games. Empfehlung - auch in vielen Foren - ist, vor Verlassen des Spiels alle Aufträge abzuschließen.

**Comment by Oberlump** [comment]
Ja, wie da vorzugehen ist, weiss ich mittlerweile - Problem ist: Mietet man wie in diesem Fall mit einem Auftrag Fahrzeuge, nutzt diese dann für mehrere Aufträge, sind die natürlich auch im Zweifel weg. Oder man vergisst es einfach abzuschließen. Da hilft besser noch: In der missions.xml zum entsprechenden Feld status auf 1 und success auf false setzen vor starten des Spiels, dann wird die Fertigstellung des Auftrages neu erkannt nach Start und alles läuft weiter.

Ich dachte aber, ich hätte vor kurzem im Changelog hier (oder wo auch immer, sorry!) etwas gelesen, dass genau dieses Problem angegangen wurde. Finde es jetzt aber auch nicht mehr.
